api/models/templates/service/redis.tmpl

```diff
@@ -2,6 +2,11 @@
   {
     "AWSTemplateFormatVersion" : "2010-09-09",
     "Parameters": {
+      "AutomaticFailoverEnabled": {
+        "Type": "String",
```

Comment:
Can we make this parameter be Yes/No like other booleans?

https://github.com/convox/rack/blob/master/api/models/templates/app.tmpl#L37-L38